### ISSUE TITLE
Exception thrown when adding fields to entity classes

### ISSUE THREAD

**Issue by DougBowen72** [JeremyLikness/SqliteWasmHelper#11]
**Describe the bug**
If you add a field to one of your entity classes it throws an exception when trying to read the cached db.

**To Reproduce**
1. Run your app such that it saves a sqlite db in the cache
2. Add a field to one of your entity classes.
3. Run the app again.
4. It attempts to select the new column from the cached sqlite db and since that new column doesn't exist it throws.

**Additional context**
Any workaround?

**Comment by Eddie-Hartman** [comment]
@JeremyLikness This question appears to be about how to handle migrations while using this project.
I'm trying to figure out how to handle doing migrations myself.
My use case being I may want to say add a column without deleting the existing data the user already has entered.
I've separated my data project into a separate csproj so I can create migrations and I'm trying to figure out how to properly apply them now.

If you have input on how you think this should work I'd appreciate the help. If I figure this out, I'll follow up here as well.

**Comment by Eddie-Hartman** [comment]
I was able to get ef migrations working.
I downloaded the SqliteWasmHelper project and added it as a reference. Then I just changed
`await ctx.Database.EnsureCreatedAsync();` in SqliteWasmDbContextFactory.cs to `await ctx.Database.MigrateAsync();`.

@JeremyLikness I would love to make this a configurable option and make a pr for it. I tried to add a boolean to the constructor called useMigrations with a default value of false to the class constructor with a condition in init so it would use EnsureCreatedAsync by default and MigrateAsync if you pass in true so I could just set that to true wherever the object is being constructed, but I couldn't figure out where exactly that would be.

I could put something up on a branch if you'd like if you think this could be easily done.
